tests/dynamic_data/users_roles/delete_user_test.py

```diff
@@ -103,6 +114,90 @@ def test_delete_user_negative(setup_users_for_testing_errors):
         assert exc_info.value.http_code == expected_status
 
 
+@pytest.mark.usefixtures('ensure_layman_module', 'oauth2_provider_mock')
+def test_delete_own_public_publications():
```

Comment:
We can parametrize with `@pytest.mark.parametrize('publication_type', process_client.PUBLICATION_TYPES)` here.